src/mlpack/bindings/python/generate_pyx.cpp.in

```diff
@@ -26,6 +26,10 @@
   #define DEBUG
 #endif
 
+#ifdef MAKING_WRAPPER
```

Comment:
Just a minor comment; it could be really helpful to add some documentation here indicating what `MAKING_WRAPPER` is and what it's for.  No need to do that immediately, but we should probably handle it before merge---it makes it a lot easier to figure out what is going on in the file without consulting the CMake configuration. :)

Reply:
I have just posted a comment where I am explaining the changes, hopefully reading that will make things clearer. Again, sorry for no comments in the code, will surely add them later.